test/source/mock/google/exported-messages/message-export-1821bf879a6f71e0.json

```diff
@@ -0,0 +1,122 @@
+          }
+        },
+        {
+          "partId": "1",
+          "mimeType": "application/octet-stream",
+          "filename": "\"what\"'s_up%3F.txt.pgp",
```

Comment:
Looks like google doesn't like this apostrophe in the middle of the file name (between `what` and `s_up`
```
Content-Disposition: attachment;
 filename*0*=utf-8''what's_up%25253F.txt.pgp
```
Probably this is why extra double quotes appear
![image](https://github.com/FlowCrypt/flowcrypt-browser/assets/23335944/3a8d0aba-3409-458a-8058-bcdb2ff3a2cd)
```
        {
          "partId": "1",
          "mimeType": "application/octet-stream",
          "filename": "\"what\"'s_up%3F.txt.pgp",
```

According to https://datatracker.ietf.org/doc/html/rfc2231#section-3
the apostrophe has special meaning like in this example
```
 title*0*=us-ascii'en'This%20is%20even%20more%20
```
we have the 2 required apostrophes right after `utf-8` denoting unspecified language, that's ok, but we should probably escape the apostrophe with %27, so we can take the filename directly from Gmail API (when decoding).
If the filename was encoded like this by the browser extension, we should also fix the encoding part.
@sosnovsky ?

Reply:
@rrrooommmaaa I just checked this message in `flowcrypt.compatibility` account and attachment has correct name in default Gmail view and in decoded FlowCrypt view:

<img width="307" alt="Screenshot 2023-05-12 at 21 39 04" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/68eb454f-ec02-400a-8df1-05741d2e1841">

<img width="230" alt="Screenshot 2023-05-12 at 21 40 31" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/993e1695-65b8-4346-8529-2786210d945b">

Not clear, what should be fixed here

Reply:
There aren't double quotes around "what" in the MIME message
```
------sinikael-?=_1-16583266435440.5429396972265759
Content-Type: application/octet-stream; name="what's_up?.txt.pgp"
Content-Disposition: attachment; filename*0*=utf-8''what's_up%3F.txt.pgp
```
My guess is that Gmail added them because the apostrophe isn't allowed in the `Content-Disposition: ...` line
We should try `Content-Disposition: attachment; filename*0*=utf-8''what%27s_up%3F.txt.pgp` instead.
Gmail also didn't decode %3F, probably because of the same reason.

Reply:
Ah, now I see that original file name is `what's_up?.txt`, initially I thought quotes were included in original file name.
Thanks for explanation, filing an issue for it